yarn.lock

```diff
@@ -1256,16 +1256,16 @@ code-point-at@^1.0.0:
   resolved "https://registry.yarnpkg.com/code-point-at/-/code-point-at-1.1.0.tgz#0d070b4d043a5bea33a2f1a40e2edb3d9a4ccf77"
   integrity sha1-DQcLTQQ6W+ozovGkDi7bPZpMz3c=
 
-codecov@^3.8.1:
-  version "3.8.2"
-  resolved "https://registry.yarnpkg.com/codecov/-/codecov-3.8.2.tgz#ab24f18783998c39e809ea210af899f8dbcc790e"
-  integrity sha512-6w/kt/xvmPsWMfDFPE/T054txA9RTgcJEw36PNa6MYX+YV29jCHCRFXwbQ3QZBTOgnex1J2WP8bo2AT8TWWz9g==
+codecov@^3.8.8:
+  version "3.8.3"
```

Comment:
Looks kinda weird that the line above says `^3.8.8` but the version installed is `3.8.3`?

Reply:
what the...how does that happen? Let me try running `yarn` again.

Reply:
fantastic catch btw!

Reply:
Mysteries of Yarn...

![shrug gif](https://media.giphy.com/media/eLvhchyvNNOuLbOtYP/giphy.gif)

Reply:
fixed, pushed and will auto-merge!